### PR TITLE
Add wait until migration kick in

### PR DESCRIPTION
**Pull request by petrkotas**
**What this PR does / why we need it**:
Once tainting the node unschedulable,
the migration can take time to kick in.

Specificaly wait for it.

**Release note**:

```release-note
NONE
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt/2564)

### CODE REVIEW

File: tests/migration_test.go [hunk 1]
```diff
@@ -1583,7 +1596,7 @@ func fedoraVMIWithEvictionStrategy() *v1.VirtualMachineInstance {
 
 func cirrosVMIWithEvictionStrategy() *v1.VirtualMachineInstance {
 	strategy := v1.EvictionStrategyLiveMigrate
-	vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.ContainerDiskFor(tests.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+	vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.ContainerDiskFor(tests.ContainerDiskAlpine), "#!/bin/bash\necho 'hello'\n")
```
Comment (SchSeba) [inline tests/migration_test.go]: you change the VM disk to Alpine but the function is `cirrosVMIWithEvictionStrategy` can you change the name also?

File: tests/migration_test.go [hunk 2]
```diff
@@ -1549,17 +1549,30 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 				By("tainting the source node as non-schedulabele")
 				tests.Taint(sourceNode.Name, "kubevirt.io/drain", k8sv1.TaintEffectNoSchedule)
 
+				By("waiting until migration kicks in")
+				Eventually(func() int {
+					migrations, err := virtClient.VirtualMachineInstanceMigration(k8sv1.NamespaceAll).List(&metav1.ListOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					runningMigrations := migrations2.FilterRunningMigrations(migrations.Items)
```
Comment (SchSeba) [inline tests/migration_test.go]: nit: can you change the import from `migrations2` to `migrations` and the object `migrations` to something else?

File: tests/migration_test.go (hunk 1 above)
Comment (petrkotas) [reply tests/migration_test.go]: @SchSeba thanks for noticing. This was an experiment, I have put i back.

File: tests/migration_test.go (hunk 2 above)
Comment (petrkotas) [reply tests/migration_test.go]: I have changed the object to `migrationList`, it is descriptive and suggest the content.

**Pull request merged**